### ISSUE TITLE
Releases not in sync, when is next tag?

### ISSUE THREAD

**Issue by AlmightyFrog** [Makuna/DFMiniMp3#94]
Hi,
I'd like to use the changes done in https://github.com/Makuna/DFMiniMp3/commit/d603f8e4f6d4cba7ad41d6c526f4cc7322940c96 for a project which is intended to be built with either arduino ide or platformio.
Not sure about, but is arduino ide latest version (0.7 as per releases/tags) not containing it yet but defined 1.1.0 as per library.properties having it?
In platformio it is as per https://registry.platformio.org/libraries/makuna/DFPlayer%20Mini%20Mp3%20by%20Makuna also 1.1.0 but two years old?

TLDR: Would you kindly make a new tagged version and maybe do them to both arduino library and platformio library compatible at once?

Thanks!

**Comment by Makuna** [comment]
v1.1.0 release was just submitted.  It may take up to 24 hours before it shows up in the Arduino library manager.
